### ISSUE TITLE
when command line include space in path the cli is not run

### ISSUE THREAD

**Issue by tyonly** [OSTC/php-sdk-binary-tools#84]
when git pull all the source in a folder with space in path,run "phpsdk-vs16-x64.bat",then display error: "不是内部或外部命令，也不是可运行的程序
或批处理文件。"

**Comment by cmb69** [comment]
Indeed, spaces in the folder name are not supported, and I don't think it makes much sense to support that, because the base folder name is supposed to have a rather short path to avoid issues with the tests (their path names may exceed PATH_MAX, and not all machines are configured to support long paths).

Anyhow, I believe this repository is no longer maintained, so consider to use https://github.com/php/php-sdk-binary-tools instead.

@dalehhirt, it might be a good idea to archive this repository, and maybe add a note that users should use https://github.com/php/php-sdk-binary-tools instead.

**Comment by tyonly** [comment]
> 

thanks,i'll use [https://github.com/php/php-sdk-binary-tools](https://github.com/php/php-sdk-binary-tools) instead
